content/en/docs/about/use-cases.md

```diff
@@ -6,34 +6,91 @@ aliases = ["/docs/", "/docs//", "/docs/usecases/"]
                     
 +++
 
-The end goal of every organization is to have their machine learning (ML) model successfully running in production  and generating value to the business. But what does it take to reach that point? Before a model ends up in production, there are potentially many steps required to build and deploy an ML model: data loading, verification, splitting, processing, feature engineering, model training and verification, hyperparameter tuning, and model serving. In addition, ML models can require more observation than traditional applications, because your data inputs can drift over time. Manually rebuilding models and data sets is time consuming and error prone. To simplify these requirements and challenges, we introduce Kubeflow.
+The end goal of every organization that utilizes machine learning (ML) is to have
+their ML model successfully running in production and generating value to the
```

Comment:
Hey @andreyvelich. That's awesome, thanks for your mega effort. 

I've just noticed something in `use-cases.md`:

- "to have ... ML model... running... and generating..."
  -> perhaps this should/could be: "to have ... ML model**s**... **run**... and **generate**..."

I'm not 100% sure about this. 

But "models" should be plural, IMHO.

@joeliedtke if you have time can you please help here? Thank you!

Reply:
That makes sense, I agree.